draft-ietf-jsonpath-base.md

```diff
@@ -1276,7 +1276,7 @@ The comparison operators `==` and `<` are defined first and then these are used
 When a path or function expression resulting in an empty nodelist appears on either side of a comparison:
 
 * a comparison using the operator `==` yields true if and only if the comparison
-is between two paths or function expressions each of which result in an empty nodelist.
+is between two expressions, each of which is a path or a function expression, and each of which result in an empty nodelist.
```

Comment:
Is a path unto itself an expression, though?

Reply:
No, you're right. `comparable` is a better term. Fixed in 6c62f1e0c8b5d48b745c2a132fb1ebcdc2494203.